userspace/files/sshd_config

```diff
@@ -68,6 +68,11 @@ TCPKeepAlive yes
 ClientAliveInterval 10
 ClientAliveCountMax 3
 
+# The low default DSCP of 0x10 lumps ssh traffic in with all other potentially
+# high bandwidth traffic. Raise the priority to ensure that this interactive traffic
+# remains responsive even when we are bandwidth limited.
+IPQoS 0xb8
```

Comment:
```suggestion
# The default DSCP of 0x10 lumps ssh traffic in with all other potentially
# high bandwidth traffic. Raise the priority to ensure that this interactive traffic
# remains responsive even when we are bandwidth limited.
IPQoS af41
```

Reply:
In the cabinet test, only 0xb8 so far maintains good responsiveness with very bad connections, while af41 is somewhat of an improvement over 0x10. Will need to test in the RAV4 to determine how bad these 2.4 GHz connections can get, perhaps this is fine for most cases.

I don't know the ramifications of setting this value obscenely high, but I'd imagine it would have no effect when not ssh'd in.